datahub-web-react/src/app/entity/shared/components/styled/search/downloadAsCsvUtil.ts

```diff
@@ -48,7 +48,12 @@ export const transformGenericEntityPropertiesToCsvRow = (
         // user owners
         properties?.ownership?.owners
             ?.filter((owner) => owner.owner.type === EntityType.CorpUser)
-            .map((owner) => (owner.owner as CorpUser).properties?.fullName)
+            .map(
+                (owner) =>
+                    (owner.owner as CorpUser).properties?.fullName ||
```

Comment:
Actually i feel the order should be reversed maybe? 

If they've set a display name, use that. If not, fallback to other information